Feature/update stale screenshots (#4241)
fleetdm/fleet: docs/01-Using-Fleet/01-Fleet-UI.md

## Scheduling queries

> The Fleet application allows you to schedule queries. This way these queries will run on an ongoing basis against the hosts that you have installed osquery on. To schedule specific queries in Fleet, you can organize these queries into "Query Packs". To view all saved packs and perhaps create a new pack, select "Schedule" from the top navigation and then click on the "Advanced" link on the top right of the page.

Reviewer: > Fleet allows you to schedule queries that run against your enrolled hosts at user-defined intervals

As part of our larger task to [reorganize the docs](https://github.com/fleetdm/fleet/issues/3931), we should start making notes of areas for improvement. For this page, we should create documentation for the schedule editor before teaching how to use query packs 
<img width="760" alt="image" src="https://user-images.githubusercontent.com/78363703/154444867-b0072680-3010-4ef9-a145-161f32f5b377.png">

> The Fleet application allows you to schedule queries. This way these queries will run on an ongoing basis against the hosts that you have installed osquery on. To schedule specific queries in Fleet, you can organize these queries into "Query Packs". To view all saved packs and perhaps create a new pack, select "Schedule" from the top navigation and then click on the "Advanced" link on the top right of the page.

Author: Yeah, I got you @mike-j-thomas. My line of thought was to update the screenshots and the associated docs so we can at least have the latest state of the UI on that part of the docs. I plan on doing another sweep on the UI docs after that :)

> The Fleet application allows you to schedule queries. This way these queries will run on an ongoing basis against the hosts that you have installed osquery on. To schedule specific queries in Fleet, you can organize these queries into "Query Packs". To view all saved packs and perhaps create a new pack, select "Schedule" from the top navigation and then click on the "Advanced" link on the top right of the page.

Reviewer: Perfect, I agree 👊🏻

> The Fleet application allows you to schedule queries. This way these queries will run on an ongoing basis against the hosts that you have installed osquery on. To schedule specific queries in Fleet, you can organize these queries into "Query Packs". To view all saved packs and perhaps create a new pack, select "Schedule" from the top navigation and then click on the "Advanced" link on the top right of the page.

Reviewer: Ah, I meant "we should create documentation for schedule editor to go before query packs on the page". I didn't mean should do it before we make improvements to the existing docs. Sorry if my comment was confusing.

![Manage Packs](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/manage-packs.png)

> If you select a pack from the list by clicking on the checkbox next to it's name, you can quickly enable and disable the entire pack, or you can click on the pack name to edit it further.

Reviewer: ```suggestion
You can enable or disable a pack by clicking the checkbox next to its name, or you can click on the pack name to edit it.
```

![Manage Packs With Pack Selected](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/manage-packs-with-pack-selected.png)

When you edit a pack, you can decide which targets you would like to execute the pack. This is a similar selection experience to the target selection process that you use to execute a new query.

![Edit Pack Targets](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/edit-pack-targets.png)

> To add queries to a pack, click on the "Add query +" link on the edit pack page. You will be presented with a modal where can take an existing scheduled query and add it to the pack. You can also define a few key details such as:

Reviewer: ```suggestion
To add queries to a pack, click on "Add query +" from the edit pack page, and select an existing query from the dropdown. From the add query modal you can also define the following options:
```

> - interval: how often should the query be executed?

Reviewer: ```suggestion
- Frequency (seconds): how often should the query be executed?
```

> - logging: which osquery logging format would you like to use?

Reviewer: ```suggestion
- Logging: which osquery logging format would you like to use?
```

Reviewer: The options below also need the correct capitalization adding, but I can't get at them from this review:

- Platform
- Minimum osquery version
- Shard (percentage)
